src/lib/mdns/Discovery_ImplPlatform.h

```diff
@@ -32,7 +33,7 @@ class ResolveDelegate
     virtual ~ResolveDelegate() {}
 };
 
-class DiscoveryManager
+class DiscoveryImplPlatform : public ServiceAdvertiser
```

Comment:
It's named as DiscoveryImplPlatform since it may also inherit future scanner interface.